Api/Endpoints/Endpoint.php

```diff
@@ -125,6 +123,7 @@ protected function requestAll($requestType, $uri, $queryParams = [], $body = [],
         }
         while ($result->getPageCount() > $page) {
             $page++;
+            /** @noinspection SlowArrayOperationsInLoopInspection */
```

Comment:
You should probably disable this inspection.

Reply:
Probably even better will be to assign `limit` and `page` directly instead of using `array_merge`